corehq/apps/app_manager/static/app_manager/js/settings/commcare_settings.js

```diff
@@ -226,6 +226,26 @@ hqDefine('app_manager/js/settings/commcare_settings', function () {
                 return setting.disabled && setting.visibleValue() !== setting['default'];
             });
 
+            // Returns values and value names as select/multiSelect options.
+            // Accepts an array of default values to support multiSelect.
+            setting.getOptions = function (values, valueNames, defaults) {
+                if (!values || !valueNames || values.length !== valueNames.length) {
+                    var msg = ("select and multiSelect widgets require values " +
+                        "and value names of equal length");
+                    console.error(msg, values, valueNames);
```

Comment:
Is it intentional to keep this?

Reply:
That's a great question! Now that values and value names are defined together, this scenario is practically impossible now, isn't it? :thinking:  I could just drop it.